data-prepper-plugins/opensearch/README.md

```diff
@@ -24,7 +24,7 @@ pipeline:
       bulk_size: 4
 ```
 
-The OpenSearch sink will reserve `otel-v1-apm-span-*` as index pattern and `otel-v1-apm-span` as index alias for record ingestion.
+The OpenSearch sink will reserve `otel-v1-apm-span-*` as index pattern and `otel-v1-apm-span` as index name for record ingestion.
```

Comment:
Using the term _alias_ here is probably correct. It creates many indices with names starting with `otel-v1-apm-span-` and then creates one index alias.

Reply:
I'm happy to keep the original wording for this and the other one below. Fixed in [30fea50](https://github.com/opensearch-project/data-prepper/pull/1690/commits/30fea50e4ca65aa2f15e335a964d438712a52b02). We do use `indexAlias` for this field in the code. 
I guess my initial thought was to keep the wording consistent through this Readme as "index name" is used to describe the configuration item `index` (a few lines below).